packages/airnode-admin/src/cli.ts

```diff
@@ -520,6 +520,13 @@ yargs
       console.log(`Airnode address: ${airnodeAddress}`);
     }
   )
+  .example([
+    ['$0 derive-airnode-address --airnode-mnemonic "test test test test test test test test test test test junk"\n'],
```

Comment:
Could you define these commands in some variable and write e2e tests for these. It would be nice to maintain the validity of those examples. (and here it's easy to do - you can skip the deployer commands since they are more challenging).